### PR TITLE
add library panels rbac

### PR DESCRIPTION
**Pull request by natellium**
Adds the library panels RBAC to the what's next. It'll be documented [here](https://github.com/grafana/grafana/pull/76480)

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 1]
```diff
@@ -128,3 +128,14 @@ _Available in private preview in Grafana Cloud_
 The Grafana Assume Role authentication provider lets Grafana Cloud users of the CloudWatch data source authenticate with AWS without having to create and maintain long term AWS Users. Using the new assume role authentication method, you no longer have to rotate access and secret keys in your CloudWatch data source. Instead, Grafana Cloud users can create an identity access and management (IAM) role that has a trust relationship with Grafana's AWS account; Grafana's AWS account will then use AWS Secure Token Service (STS) to create temporary credentials to access the user's AWS data.
 
 To learn more, refer to the [CloudWatch authentication documentation](/docs/grafana/next/datasources/aws-cloudwatch/aws-authentication).
+
+## RBAC for library panels
+
+<!-- Kay Delaney, Natalia Bernarte  -->
+<!-- Enterprise, Cloud -->
+
+_Available in public preview in Grafana Enterprise and Generally available in Grafana Cloud_
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:137]: I'll add the specifics of the Enterprise availability when I copy this over to the on-prem What's new. Also, we want to add a date in the Cloud version of this; is it just the 10.2 release date?
```suggestion
<!-- Enterprise public preview -->

<Date>

_Generally available in Grafana Cloud_
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 2]
```diff
@@ -128,3 +128,14 @@ _Available in private preview in Grafana Cloud_
...
+
+You can manage the permissions for library panels with [RBAC](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/access-control/) which provides a way to extend the permissions avaiable with basic roles and enables you more granular control of users' actions.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:139]:
```suggestion
You can manage the permissions for library panels with [RBAC](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/), which provides a way to extend the permissions available with basic roles and gives you more granular control of users' actions.
```

**Review by imatwawana — COMMENTED** [review]
Made some edits and left a question.

File: docs/sources/whatsnew/whats-new-next/index.md (hunk 1 above)
Comment (natellium) [reply docs/sources/whatsnew/whats-new-next/index.md:137]: It's just 10.2

**Comment by natellium** [comment]
@imatwawana Added your changes. This will go with the 10.2 release so don't think we need a specific date?

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 3]
```diff
@@ -128,3 +128,16 @@ _Available in private preview in Grafana Cloud_
 The Grafana Assume Role authentication provider lets Grafana Cloud users of the CloudWatch data source authenticate with AWS without having to create and maintain long term AWS Users. Using the new assume role authentication method, you no longer have to rotate access and secret keys in your CloudWatch data source. Instead, Grafana Cloud users can create an identity access and management (IAM) role that has a trust relationship with Grafana's AWS account; Grafana's AWS account will then use AWS Secure Token Service (STS) to create temporary credentials to access the user's AWS data.
 
 To learn more, refer to the [CloudWatch authentication documentation](/docs/grafana/next/datasources/aws-cloudwatch/aws-authentication).
+
+## RBAC for library panels
+
+<!-- Kay Delaney, Natalia Bernarte  -->
+<!-- Enterprise public preview -->
+
+<Date>
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:137]:
```suggestion
October 24, 2023
```

**Comment by imatwawana** [comment]
> @imatwawana Added your changes. This will go with the 10.2 release so don't think we need a specific date?

Because it's not a given for someone reading Cloud docs that this was part of 10.2, and because other Cloud features aren't tied to the Grafana release, we're adding dates for all entries.

**Comment by natellium** [comment]
Ahh, got it! Makes sense.

**Comment by imatwawana** [comment]
@natellium - Please don't merge this until @jdbaldry  has had a look. I'm not sure why the version interpolation syntax isn't working.

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 4]
```diff
@@ -128,3 +128,16 @@ _Available in private preview in Grafana Cloud_
...
+## RBAC for library panels
+
+<!-- Kay Delaney, Natalia Bernarte  -->
+<!-- Enterprise public preview -->
+
+October 24, 2023
+
+_Generally available in Grafana Cloud_
+
+You can manage the permissions for library panels with [RBAC](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/), which provides a way to extend the permissions available with basic roles and gives you more granular control of users' actions.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:141]:
```suggestion
You can manage the permissions for library panels with [RBAC](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/roles-and-permissions/access-control/), which provides a way to extend the permissions available with basic roles and gives you more granular control of users' actions.
```

**Comment by imatwawana** [comment]
> @natellium - Please don't merge this until @jdbaldry has had a look. I'm not sure why the version interpolation syntax isn't working.

Version syntax fixed. This is good to go!

**Review by imatwawana — APPROVED** [review]

**Pull request merged**